src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -176,7 +186,11 @@ function startWatchingForValueChange(target, targetID) {
   );
 
   Object.defineProperty(activeElement, 'value', newValueProp);
-  activeElement.attachEvent('onpropertychange', handlePropertyChange);
+  if (activeElement.attachEvent) {
```

Comment:
yeah, again more defensive than anything, I _know_ this event works with attachEvent in browsers that support it per the docs and experience, whereas with addEventListener, I only confirm it works in ie11 since that's all I have access too (that despite the ambiguity of the MS docs on support). 

It may be that its supported in IE11 just because `attachEvent` doesn't exist anymore.